core/modules/qdisp/Executive.h

```diff
@@ -157,6 +162,7 @@ class Executive : public std::enable_shared_from_this<Executive> {
     XrdSsiService* _xrdSsiService; ///< RPC interface
     JobMap _jobMap; ///< Contains information about all jobs.
     JobMap _incompleteJobs; ///< Map of incomplete jobs.
+    std::atomic<int> _totalJobs{1}; ///< How many jobs are used in this query.
```

Comment:
It's set to 1 so that there isn't a 0 out of 0 chunks completed possibility being sent to the user when the worker queries are being generated. The value is overwritten by the correct value when that value is know.

Reply:
Can you add a comment saying that it should be initialized with non-zero value?